varinfo/generate_umm_var.py

```diff
@@ -36,25 +36,28 @@ def generate_collection_umm_var(collection_concept_id: str,
         * collection_concept_id: Concept ID for collection that variables have
           been generated for.
         * cmr_env: URLs for CMR environments (OPS, UAT, and SIT)
-        * auth_header: Authorization HTTP header, containing a LaunchPad
-          token: 'Authorization: <token>'
+        * launchpad_token: A LaunchPad token with no header prefixes:
```

Comment:
Just carrying on from my previous comment - per the logic in `get_edl_token_header`, technically we are also still allowing authorisation headers with an EDL bearer token in them.